SETUP/db_schema.sql

```diff
@@ -360,6 +360,7 @@ CREATE TABLE `projects` (
   `language` varchar(255) NOT NULL default '',
   `username` varchar(25) NOT NULL default '',
   `comments` text NOT NULL,
+  `unsafe_comment` tinyint(1) NOT NULL default '0',
```

Comment:
It might be better to have this be `comment_format varchar(8)` and default to `markdown` with `html` being the grandfathered version. This gives us a bit more flexibility if we need to support a different type in the future and also is clearer about what the comment format is. "unsafe" to me is ambiguous and could mean that we should be `html_safe()`ing the comment field if true, which is actually the opposite of what we want.

Reply:
completely agree. I knew I'd get feedback here. Thanks.